packages/next-codemod/transforms/next-lint-to-eslint-cli.ts

```diff
@@ -0,0 +1,822 @@
+import { existsSync, readFileSync, writeFileSync } from 'node:fs'
+import path from 'node:path'
+import { getPkgManager, installPackages } from '../lib/handle-package'
+import { createParserFromPath } from '../lib/parser'
+
+interface TransformerOptions {
+  skipInstall?: boolean
+  [key: string]: unknown
+}
+
+const ESLINT_CONFIG_TEMPLATE_TYPESCRIPT = `import { dirname } from "path";
```

Comment:
non-blocking: if you want to start code in new line but without line break you can do:
```js
const ESLINT_CONFIG_TEMPLATE_TYPESCRIPT = `\
import { dirname } from "path";
```